### ISSUE TITLE
Unable to agro Archaic Chests in uncapped Psojax

### ISSUE THREAD

**Issue by Fng098** [EdenServer/community#5243]
### Checklist

- [X] I have searched for existing issues for issues like this one. The issue has not been posted.
- [X] I have provided reproducible steps. (No "as the title says" posts please. ***Please try to state facts only, and avoid assumptions/opinions.***)
- [X] I have provided my client version in the details. (type /ver into your game window)
- [X] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details
Version 30181205_0

Unable to agro Archaic Chests in uncapped Psojax at I-6 and G-6.  Unable to agro by touching chest, or trading living key.  Do have quest open in quest log.

**Comment by ConteAlmaviva** [comment]
Probably not the answer you're looking for, but something you could try would be to pull something else and cast an AoE like Diaga, which usually wakes up the chest if it's a mimic.

**Comment by Fng098** [comment]
Yeah I'm dual boxing a THF and a rdm. Doubt I can kill both the mimic and the elemental.

**Comment by randomslumber** [comment]
Still an issue. Only thing that we were able to use on the mimic to aggro it was ranged attacks.
